hip-0000-template.md

```diff
@@ -1,59 +1,276 @@
+IPFS, as a core protocol, offers robust immutability by design, making it an ideal choice for storing critical data related to DEAs. This immutability means that once data is stored on IPFS, it cannot be altered or tampered with, ensuring the authenticity and reliability of the information associated with these assets.
+
+**Beware of Mutable Data Using IPNS**
+
+While IPFS guarantees immutability, IPNS introduces the possibility of data mutation. IPNS allows for the dynamic updating of content associated with a given CID (Content Identifier) by mapping it to different content over time. This capability, while useful in certain contexts, poses a potential security risk for the underlying DEAs. Mutable data can lead to inconsistencies, loss of trust, and challenges in verifying the historical accuracy of an asset's metadata.
+To maintain the highest level of security and data integrity, it is strongly recommended to avoid using IPNS for DEA metadata. Instead, adhere to the principles of immutability provided by IPFS, or protocols built on top, itself. This ensures that once data is recorded and associated with an asset, it remains unaltered and reliable throughout its lifecycle.
 
 ## How to Teach This
 
-For a HIP that adds new functionality or changes interface behaviors, it is helpful to include a section on how to teach users, new and experienced, how to apply the HIP to their work.
+Guardian-compliant HTS implementations use this standard to provide additional metadata for their DEAs.
+
+Wallet and token explorer implementations interrogate HTS tokens using this standard to display additional metadata for tokens.
 
 ## Reference Implementation
 
-The reference implementation must be complete before any HIP is given the status of “Final”. The final implementation must include test code and documentation.
+This section provides examples of a metadata schema adhering to this specification.
+
+### Example Schema: Required and Recommended fields ONLY
+
+```json
+{
+    "name": "Redhill Farm Carbon Credit",
+    "creator": "Redhill Farm",
+    "format": "guardian-dea",
+    "description": "This is a carbon credit issued by Redhill Farm",
+    "image": "ipfs://bafybeiapq5ewrpxrsche54laxvwwvth2ye3df4wbswwe3z2mk7rgzlz7za",
+    "type": "image/png",
+    "properties": {
+        "guardian_proof": {
+            "type": "Ed25519Signature2018",
+            "created": "2023-10-04T11:18:59Z",
+            "verificationMethod": "did:hedera:testnet:Ci3QBHbe3Fa6gKn8hm44PJn1uMkXEDbJkPQboYN7EKWV_0.0.3160930#did-root-key",
+            "proofPurpose": "assertionMethod",
```

Comment:
HIP412 tends to use snake case instead of camel. Apart from `creatorDid` which slipped through the net! 🙈 

Suggest we continue with snake case throughout.

```suggestion
            "verification_method": "did:hedera:testnet:Ci3QBHbe3Fa6gKn8hm44PJn1uMkXEDbJkPQboYN7EKWV_0.0.3160930#did-root-key",
            "proof_purpose": "assertionMethod",
```

Reply:
non-issue on these lines, this is is related to the W3C standards format for [verifiable credentials](https://www.w3.org/TR/vc-data-model/#concrete-lifecycle-example). Understand the creatorDid but as that is in spec I dunno if it is worth changing here.